CONTRIBUTING.md

```diff
@@ -15,6 +19,32 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Data file management
+
+### Setup directory
+
+Each module should contain a `setup` directory that includes instructions and scripts to download and prepare any input data files used by the notebooks in the module.
+
+### Linking shared files 
+
+When trainings are run from the RStudio server (rstudio.ccdatalab.org), we store large input data files in the `/shared/data/training-modules` directory.
+The organization within this directory should mirror the arrangement of the repository, so that we can easily link files and folders from this shared directory to mirrored paths within a clone of this repository.
+
+Linking files from the shared directory to a cloned repository is done with the `scripts/link-data.sh` bash script, so this script should be kept up to date as any needed files are added to the `/shared/data/training-modules` directory.
+When possible, link to enclosing directories rather than individual files to keep links simpler, but see an important caveat below.
+
+Because this script is also used to set up directories for training, the links should not include _all_ files needed for _every_ notebook: 
+- Files that are created during a training module should not  be included in this script
```

Comment:
Not sure the live is worth specifying....

```suggestion
- Files that are created during a live training module should not be included in this script.
```

Reply:
How about `training session`?

Reply:
Sounds good!